.circleci/config.yml

```diff
@@ -128,15 +128,15 @@ jobs:
       - run:
           name: Integration tests (Autopush Legacy)
           command: make integration-test-legacy
+          environment:
+            TEST_RESULTS_DIR: workspace/test-results
       - run:
           name: Integration tests (Autoconnect)
           command: make integration-test
-      - save_cache:
-          name: Save Python cache
-          key: python-v1-{{ checksum "tests/poetry.lock" }}-{{ checksum "tests/integration/test_integration_all_rust.py"}}
-          paths:
-            - /home/circleci/.local/bin/
-            - /home/circleci/.local/lib/
+          environment:
+            TEST_RESULTS_DIR: workspace/test-results
```

Comment:
Maybe these are here intentionally to be explicit but I'll note they match the default, so they're not necessary